listcmd/list.go

```diff
@@ -0,0 +1,123 @@
+package listcmd
```

Comment:
Just a suggestion, `listcmd` looks like the List implementation at first sight, maybe we can change it to sth else. In future, we may want to implement the actual `listcmd` for the List data structure and it can create conflict.

Reply:
Good idea! Renamed it https://github.com/hazelcast/hazelcast-commandline-client/pull/133/commits/9708369995df3bc8f2e73ac0358270d752507e50